tools/wptrunner/wptrunner/testloader.py

```diff
@@ -430,8 +431,7 @@ def update_manifest(self, manifest_path, tests_path, url_base="/",
 
         manifest.write(manifest_file, manifest_path)
 
-    def load_manifest(self, tests_path, metadata_path, url_base="/"):
-        manifest_path = os.path.join(metadata_path, "MANIFEST.json")
+    def load_manifest(self, tests_path, manifest_path, url_base="/", **kwargs):
```

Comment:
What's the point in changing this to accept `**kwargs`?

Reply:
It's populated from the dictionary we build but only cares about some keys. Otherwise we would either need to change the caller or have some explicit unused parameters.

Reply:
This change broke the Servo update process:
```
TypeError: load_manifest() takes at least 3 arguments (3 given)

  File "/Users/servo/buildbot/slave/mac-nightly/build/python/servo/testing_commands.py", line 453, in update_wpt
    return run_globals["update_tests"](**kwargs)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/update.py", line 25, in update_tests
    rv = update.run_update(logger, **kwargs)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/update/__init__.py", line 26, in run_update
    return updater.run()
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/update.py", line 151, in run
    rv = update_runner.run()
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/base.py", line 61, in run
    rv = step(self.logger).run(step_index, self.state)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/base.py", line 32, in run
    self.create(state)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/update.py", line 97, in create
    runner.run()
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/base.py", line 61, in run
    rv = step(self.logger).run(step_index, self.state)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/base.py", line 32, in run
    self.create(state)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/update/metadata.py", line 37, in create
    stability=state.stability)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/metadata.py", line 40, in update_expected
    manifests = load_test_manifests(serve_root, test_paths)
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/metadata.py", line 27, in load_test_manifests
    return manifest_loader.load()
  File "/Users/servo/buildbot/slave/mac-nightly/build/tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/testloader.py", line 394, in load
    **paths)
```